proposals/0107-unsaferawpointer.md

```diff
@@ -1460,15 +1463,15 @@ even encourages interoperability between C APIs and a String's UTF8
 encoding. For example:
 
 ```swift
-  var utf8 = template.nulTerminatedUTF8
-  let (fd, fileName) = utf8.withUnsafeMutableBufferPointer {
-    (utf8) -> (CInt, String) in
-    let cStrBuf = UnsafeRawPointer(utf8.baseAddress!)
-      .cast(to: UnsafePointer<CChar>)
-    let fd = mkstemps(cStrBuf, suffixlen)
-    let fileName = String(cString: cStrBuf)
-    ...
-  }
+var utf8 = template.nulTerminatedUTF8
+let (fd, fileName) = utf8.withUnsafeMutableBufferPointer {
+  (utf8) -> (CInt, String) in
+  let cStrBuf = UnsafeRawPointer(utf8.baseAddress!)
+    .assumingMemoryBound(to: UnsafePointer<CChar>)
```

Comment:
@atrick I figured `cast(to:)` was an earlier name for `assumingMemoryBound(to:)`—is that correct?